tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
   "compilerOptions": {
     "jsx": "react",
-    "target": "es5",
+    "target": "es6",
     "module": "esnext",
     "moduleResolution": "node",
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
     "noImplicitAny": true,
+    "skipLibCheck": true,
```

Comment:
Just curious, why did you need to add this?

Reply:
Some of the dependencies use lodash and do not have proper types.